client/src/components/ElementEditor/tests/ElementActions-test.js

```diff
@@ -10,26 +10,24 @@ import AbstractAction from 'components/ElementActions/AbstractAction';
 Enzyme.configure({ adapter: new Adapter() });
 
 describe('ElementActions', () => {
-  const ActionMenuComponent = (props) => <div>{props.children}</div>;
-  const testTabs = [
-    { title: 'Content', name: 'Main' },
-    { title: 'Settings', name: 'Settings' },
-    { title: 'History', name: 'History' }
-  ];
-  const editTabsClick = () => {};
+  let props = {};
+  beforeEach(() => {
```

Comment:
I did some refactoring on the unit test cos there was a lot of code duplication between the existing tests